src/docs/user/references/Notification-service.adoc

```diff
@@ -0,0 +1,237 @@
+=== Overview
+
+The <<_glossary_notification_portal,*Notification Portal*>> allows users to subscribe, be notified and react when different
+types of events occur during <<_glossary_job,*Job*>> execution, on the state of the <<_glossary_proactive_scheduler,*ProActive Scheduler*>> or when
+automating and scheduling jobs with the <<_glossary_job_planner,*Job Planner*>>.
```

Comment:
or when
automating and scheduling jobs with the <<_glossary_job_planner,*Job Planner*>>.   ??

Reply:
Please rephrase, this sentence is too long and not understandable. Split into several sentences.